### ISSUE TITLE
[codserver] can't find pane cod-server

### ISSUE THREAD

**Issue by dgibbs64** [GameServerManagers/LinuxGSM#1167]
Im trying to get codserver working however running in to this issue when trying to start it

```
can't find pane cod-server
[ FAIL ] Starting cod-server: Unable to start LinuxGSM
```

https://github.com/GameServerManagers/LinuxGSM/tree/cod1

**Comment by cedarlug** [comment]
I'm seeing a lot of issues with a fresh installation. 
There's nothing in the serverfiles directory, and specifically no server.cfg.

http://hastebin.com/toqisovaji.sql

Digging deeper.

**Comment by cedarlug** [comment]
If this helps, here's what I've found with a fresh install using the cod1 branch:

The initial game refuses to start because it's looking for `~/lgsm/lib/libgcc_s.so.1`
and `~/lgsm/lib//libstdc++.so.5.0.3`.

These aren't found because they're actually residing in `~/lgsm/lib/gcc3/`  (note the `gcc3` directory is where they are located, but isn't being searched.  There's a ubuntu directory there as well).

Further, a wget for these files from gameservermanagers fails:
```
[ .... ] Starting cod-server:     fetching libstdc++.so.5.0.3...FAIL
https://raw.githubusercontent.com/GameServerManagers/LinuxGSM/master/lgsm/lib/gcc3/libstdc++.so.5.0.3
```

Putting a symlink in `~/lgsm/lib` to the files in `~/lgsm/lib/gcc3` worked to fully bootstrap the server.

**Comment by dgibbs64** [comment]
I was not aware of the gcc3 directory. I will have a look in to this one. It might help thanks

**Comment by dgibbs64** [comment]
@cedarlug the server is starting for me. The Ubuntu directory was added by me for glibc fixes. However gcc3 was added by  @hurdad I will see if I can get this working.

**Comment by cedarlug** [comment]
@dgibbs64  Did you mv/symlink the lib's into lgsm/lib or was there another solution?

**Comment by dgibbs64** [comment]
I have just pushed a change :) https://github.com/GameServerManagers/LinuxGSM/commit/1608d2a4b40291ebf23de2b8c9bab9eec3bdb36f

**Comment by dgibbs64** [comment]
Issue now resolved cab1cef5e5541c0e3de84281c073c2cef0fcbdf1
